iphone/Maps/Maps.xcodeproj/project.pbxproj

```diff
@@ -1764,6 +1767,7 @@
 		29B97323FDCFA39411CA2CEA /* Frameworks */ = {
 			isa = PBXGroup;
 			children = (
+				EBA1BC3C2A3BD69E00E33F7A /* libfast_obj.a */,
```

Comment:
Isn't it a header-only library? Can it be included directly where necessary, in one of our source files?

Reply:
Yes, it can be a header only. 

> Can it be included directly where necessary, in one of our source files?
Up to you